src/main/java/lotto/service/LottoGame.java

```diff
@@ -0,0 +1,38 @@
+package lotto.service;
+
+import lotto.domain.AutoLotto;
+import lotto.domain.Lotto;
+import lotto.domain.LottoNum;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoGame {
+    public static final int LOTTO_PRICE = 1000;
+
+    private LottoGame() {
+    }
+
+    private static LottoGame INSTANCE = null;
+
+    public static LottoGame getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new LottoGame();
+        }
+        return INSTANCE;
+    }
+
+    public List<Lotto> buy(int priceTotal) {
+        if (priceTotal < LOTTO_PRICE) {
+            throw new IllegalStateException("금액이 부족합니다.");
+        }
+        List<Lotto> result = new ArrayList<>();
+        for (int i = 0; i < priceTotal / LOTTO_PRICE; i++) {
+            result.add(AutoLotto.getInstance()
+                    .getLotto(LottoNum.getLottoNumbers(), Lotto.LOTTO_NUMBER_COUNT));
```

Comment:
```suggestion
            result.add(AutoLotto.getInstance().getLotto());
```
이 부분도 심플하게 가져가도 될 것 같아요

Reply:
수정했습니다